add-on/panels/js/slideshow.jsx

```diff
@@ -0,0 +1,281 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var loop = loop || {};
+loop.slideshow = (function(_, mozL10n) {
+  "use strict";
+
+// App state
```

Comment:
Please can you extract the slideshow part of this that was copied from the other repo, and put it in a file in add-on/panels/vendor. At the top of the file, please reference the source, and add their license (see urlRegExps.js for an example of where we've done this before).

In this file, we can keep the init and the data structure, and keep the mpl license.